spring-cloud-starter-stream-source-http/src/main/java/org/springframework/cloud/stream/app/http/source/HttpSourceConfiguration.java

```diff
@@ -46,7 +42,7 @@
  * @author Christian Tzolov
  */
 @EnableBinding(Source.class)
-@EnableConfigurationProperties(HttpSourceProperties.class)
+@EnableConfigurationProperties({ HttpSourceProperties.class, SecurityCommonAutoConfigurationProperties.class })
```

Comment:
And why do you include this `SecurityCommonAutoConfigurationProperties` here?
Isn't it included automatically by the `SecurityCommonAutoConfiguration`, so then it must not be here on the HTTP Source level.

Reply:
+1